src/component/parent/drivers.js

```diff
@@ -150,6 +150,7 @@ export let RENDER_DRIVERS = {
 
             this.iframe.style.zIndex   = MAX_Z_INDEX;
             this.iframe.style.position = 'fixed';
+            this.iframe.className  = 'xcomponent-lightbox bounceInUp';
```

Comment:
Can we avoid hard-coding `bounceInUp` here? I'm not sure every component is going to want this behavior. It might be best just to keep the `xcomponent-lightbox` class and have the component add the bounce CSS under that class.